### PR TITLE
DOC for Unhide and Hide commands in Uniconfig shell

### PR DESCRIPTION
**Pull request by ltokarcik**

### CODE REVIEW

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 1]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
 List of invoked commands are persisted across UniConfig restarts and SSH connections.
 !!!
 
+### Unhide and Hide operations
+
+* Command 'unhide-get' is used to unhide attribute that was hidden in application properties for read.
+* Command 'unhide-set' is used to unhide attribute that was hidden in application properties for write.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:148]: an attribute

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 2]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+* Command 'hide-get' is used to hide again unhidden attribute with 'unhide-get' command.
+* Command 'hide-set' is used to hide again unhidden attribute with 'unhide-set' command.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:150]: remove 'again'

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 3]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+
+* When unhide is set for GET or SET operation, then request URL for this operation contains 'unhide' query parameter like bellow:  
+  http://localhost:8181/rests/data/network-topology:network-topology/topology=uniconfig/node=vnf21/configuration?unhide=all  
+  (In this example is unhide parameter set to 'all')
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:154]: "In this example, the unhide parameter is set to 'all' "

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 4]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+
+!!!
+Command also confirm that attribute was added or removed from unhidden list.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:157]: The command also confirms that the attribute

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 5]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+When is 'unhide-get' or 'hide-get' called without parameter, then output contains list of all unhidden parameters.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:158]: When 'unhide-get' or 'hide-get' is called without a parameter, then the output contains a list ....

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 6]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+It is simillar also for 'unhide-set' and 'hide-set' commands.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:159]: It is also similar to 'unhide-set' and 'hide-set' commands.

File: frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md [hunk 7]
```diff
@@ -138,6 +142,49 @@ config>show-history 5
...
+unhidden get:
+tailf:hidden deprecated
+tailf:hidden debug
+uc>unhide-set
+unhidden set:
+tailf:hidden full
+```
+
+!!!
+There could be used also parameter all, that unhide all parameters for read or for write that are defined in application properties.
```
Comment (jbogusciak) [inline frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md:185]: The 'all' parameter could also be used, which unhides all parameters for read or for write that are defined in application properties.

**Review by jbogusciak — CHANGES_REQUESTED** [review]

**Review by knuuzor — COMMENTED** [review]
Copy check OK

**Pull request merged**